test/libsolidity/semanticTests/array/create_memory_array.sol

```diff
@@ -16,6 +16,7 @@ contract C {
         return (x[199], y[203][1], z[170].a[1], z[170].b[99]);
     }
 }
-
+// ====
+// compileViaYul: also
```

Comment:
Why didn't this fail the "force yul" test?

Reply:
that's a good question. @aarlt I think you wrote the force yul tests, any idea?

Reply:
@Marenz I didn't implemented this.. I think it was @mijovic

Reply:
sorry my bad @aarlt :D

Reply:
I wrote force-yul. But not sure why this test passes. Which feature is not implemented in this test?

Reply:
I think all features are implemented which is what prompted the question :)